tool/speed.cc

```diff
@@ -1022,6 +1022,33 @@ static bool SpeedECDHCurve(const std::string &name, int nid,
   return true;
 }
 
+static bool SpeedECGenCurve(const std::string &name, int nid,
```

Comment:
We are generating a key, not a curve:
```suggestion
static bool SpeedECKeyGenCurve(const std::string &name, int nid,
```

Reply:
This should be read as EC Key-Gen Curve (i.e. Generate a key for the specific curve, rather than generate a curve). It's a bit confusing but I was trying to stay consistent with how our other EC tests behave/are named

Reply:
not sure I get it, I'm suggesting to to rename the function to `SpeedECKeyGenCurve` which would mean exactly what you are saying: EC KeyGen for a Curve.